### PR TITLE
Bump Haskell Language Server to 1.8.0.0

### PR DESCRIPTION
**Pull request by zeme-iohk**
We can't seem to get 1.9 but 1.8 seems to work

### CODE REVIEW

File: __std__/cells/plutus-apps/library/make-plutus-apps-project.nix [hunk 1]
```diff
@@ -24,7 +24,7 @@ let
       "https://github.com/input-output-hk/cardano-config"."1646e9167fab36c0bff82317743b96efa2d3adaa" = "sha256-TNbpnR7llUgBN2WY7CryMxNVupBIUH01h1hRNHoxboY=";
       "https://github.com/input-output-hk/cardano-ledger"."da3e9ae10cf9ef0b805a046c84745f06643583c2" = "sha256-3VUZKkLu1R43GUk9IwgsGQ55O0rnu8NrCkFX9gqA4ck=";
       "https://github.com/input-output-hk/cardano-wallet"."18a931648550246695c790578d4a55ee2f10463e" = "0i40hp1mdbljjcj4pn3n6zahblkb2jmpm8l4wnb36bya1pzf66fx";
-      "https://github.com/Quviq/quickcheck-contractmodel"."462f8c25bea15f88e179e12ebf4e3a4ffcc92d18" = "sha256-0NXBaWLGL5Hoz/A/yLIh5NZ4F5tqQUScB4Tr5u1Wg1Y=";
+      "https://github.com/Quviq/quickcheck-contractmodel"."5cd16ba995f1cd48c2b5f885c702ca6f2a2abbaa" = "sha256-rVVdr68f+xEOV4s27Ix+wdBznSVFvq+Q2ZBo0SEnkp0=";
```
Comment (zeme-iohk) [inline __std__/cells/plutus-apps/library/make-plutus-apps-project.nix:27]: this is just catching up the non-std project derivation

Comment (kayvank) [reply __std__/cells/plutus-apps/library/make-plutus-apps-project.nix:27]: @zeme-iohk on my system, this PR seem to be rebuilding, seems to be world` :) Is that to be expected? I am on nix-os and here is my /etc/nix/nix.conf:

 plutus-apps git:(zeme-iohk/std-4) ✗ cat /etc/nix/nix.conf | pbcopy
```
# WARNING: this file is generated from the nix.* options in
# your NixOS configuration, typically
# /etc/nixos/configuration.nix.  Do not edit it!
allowed-users = *
auto-optimise-store = true
builders = 
cores = 0
extra-sandbox-paths = 
max-jobs = auto
require-sigs = true
sandbox = true
sandbox-fallback = false
substituters = https://cache.nixos.org/ https://cache.iog.io https://cache.zw3rk.com https://cache.nixos.org/
system-features = nixos-test benchmark big-parallel kvm
trusted-public-keys = cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= loony-tools:pr9m4BkM/5/eSTZlkQyRt57Jz7OMBxNSUiMC4FkcNfk=
trusted-substituters = 
trusted-users = root kayvan
experimental-features = nix-command flakes
keep-outputs          = true
keep-derivations      = true

**Review by michaelpj — APPROVED** [review]
Do you want to add a comment explaining why we can't get 1.9? I think it's some kind of weird compiler incompatibility, so it should be possible once plutus-apps hits GHC 9.2, so worth putting that down.

File: __std__/cells/plutus-apps/library/make-plutus-apps-project.nix (hunk 1 above)
Comment (zeme-iohk) [reply __std__/cells/plutus-apps/library/make-plutus-apps-project.nix:27]: Are you sure it's the World? It should only be building HLS proper. Your config seems fine also..

**Pull request merged**